src/components/KDateTimePicker/KDateTimePicker.vue

```diff
@@ -693,7 +707,15 @@ $grid-spacing: var(--kui-space-30, $kui-space-30);
 
 // v-calendar overrides
 .k-datetime-picker {
-  $highlight-color: var(--kui-color-background-primary-weaker, $kui-color-background-primary-weaker);
+  // Use `Inter` font throughout
+  // stylelint-disable-next-line
+  --vc-font-family: var(--kui-font-family-text, #{$kui-font-family-text});
+
+  // All calendar dates use v-calendar's small text variable
+  // stylelint-disable-next-line
+  --vc-text-sm: var(--kui-font-size-20, #{$kui-font-size-20});
+
+  $highlight-color: var(--kui-color-background-primary-weakest, $kui-color-background-primary-weakest);
```

Comment:
issue: use interpolation when setting a SCSS variable
```suggestion
  $highlight-color: var(--kui-color-background-primary-weakest, #{$kui-color-background-primary-weakest});
```

Reply:
The line above it (`--vc-text-sm`) is a Custom Property, but this line is just a local variable